cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -124,6 +128,10 @@ void Run(const std::vector<string>& bag_filenames) {
     static_tf_broadcaster.sendTransform(urdf_transforms);
   }
 
+  ros::AsyncSpinner async_spinner(1, ros::getGlobalCallbackQueue());
```

Comment:
Fixed, also added a constant which clarifies the number of threads for the `AsyncSpinner`.